### PR TITLE
lasi: fix shim path pollution

### PR DESCRIPTION
**Pull request by mitchblank**
This failed auditing due to:
```
  * Files were found with references to the Homebrew shims directory.
    The offending files are:
      share/lasi1.1.3/examples/Makefile
```
This is basically the same issue I had with plplot's Makefiles.examples (#66985)  Similar fix works.

### CODE REVIEW

**Comment by mitchblank** [comment]
Looks like the 10.14 CI failure was just a GH timeout:
```
Run Homebrew/actions/setup-homebrew@master
Reset branch 'master'
Branch 'master' set up to track remote branch 'master' from 'origin'.
Your branch is up to date with 'origin/master'.
From https://github.com/Homebrew/homebrew-core
 * branch                    9aacef5e16421e0f04c7c3d4a417c9dad05a174f -> FETCH_HEAD
fatal: unable to access 'https://github.com/Homebrew/homebrew-core/': transfer closed with outstanding read data remaining
Error: Process completed with exit code 128.
```

File: Formula/lasi.rb [hunk 1]
```diff
@@ -27,7 +28,21 @@ def install
     # None is valid, but lasi's CMakeFiles doesn't think so for some reason
     args = std_cmake_args - %w[-DCMAKE_BUILD_TYPE=None]
```
Comment (fxcoudert) [inline Formula/lasi.rb:29]: `std_cmake_args` contains `-DCMAKE_BUILD_TYPE=Release` nowadays, not `-DCMAKE_BUILD_TYPE=None`, so this all thing can be removed.

File: Formula/lasi.rb [hunk 2]
```diff
@@ -27,7 +28,21 @@ def install
...
 
+    # std_cmake_args tries to set CMAKE_INSTALL_LIBDIR to a prefix-relative
+    # directory, but plplot's cmake scripts don't like that
+    args.map! { |x| x.start_with?("-DCMAKE_INSTALL_LIBDIR=") ? "-DCMAKE_INSTALL_LIBDIR=#{lib}" : x }
+
     system "cmake", ".", "-DCMAKE_BUILD_TYPE=Release", *args
+
+    inreplace "examples/Makefile.examples" do |s|
```
Comment (fxcoudert) [inline Formula/lasi.rb:37]: Let's just not install the examples, then.

**Review by chenrui333 — APPROVED** [review]
👍

**Pull request closed without merging**